src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/FaultedTreeCommandGroupHandler.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.VisualStudio.Input;
+using Microsoft.VisualStudio.ProjectSystem.Input;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Input.Commands
+{
+    internal class FaultedTreeCommandGroupHandler : IAsyncCommandGroupHandler
+    {
+        [ExportCommandGroup(CommandGroup.VisualStudioStandard97)]
+        [AppliesTo(ProjectCapability.CSharpOrVisualBasic)]
+        [Order(int.MaxValue)] // We want to be ahead of the standard handlers
+        public static IAsyncCommandGroupHandler VisualStudioStandard97Handler =>
+            new FaultedTreeCommandGroupHandler(new List<long> { VisualStudioStandard97CommandId.UnloadProject });
+
+        [ExportCommandGroup(CommandGroup.VisualStudioStandard2k)]
+        [AppliesTo(ProjectCapability.CSharpOrVisualBasic)]
+        [Order(int.MaxValue)] // We want to be ahead of the standard handlers
+        public static IAsyncCommandGroupHandler VisualStudioStandard2kHandler =>
```

Comment:
Well, I figured it would be the easiest implementation, so I tried it, and it worked just fine.